libs/web/feature-auth/src/lib/hooks/useLogin.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState, useEffect } from 'react';
+import { useRouter } from 'next/router';
+import { validateEmail } from '@daruma-board/web/design-system';
+
+
+
+export const useLogin = () => {
+    const [email, setEmail] = useState<string>('');
+    const [auth, setAuth] = useState<boolean>(false); // [auth, setAuth
+    const [loading, setLoading] = useState<boolean>(false); // [loading, setLoading
+    const router = useRouter();
+
+
+    const handleLogin = async (email: string) => {
+      setLoading(true);
+
+      if (!email) {
+        alert('Email is required');
+        setLoading(false);
+        return;
+      }
+  
+      if (!validateEmail(email)) {
+        alert('Email is invalid');
+        setLoading(false);
+        return;
+      }
+  
+      await fetch('/api/auth-email', {
+        method: 'POST',
+        body: JSON.stringify({ email }),
+      })
+        .then((res) => {
+          if (res.status === 200) {
+            console.log(res);
```

Comment:
Nit: remove console log

Reply:
Sorry just saw this before i made the most recent pushed. I will re commit